### ISSUE TITLE
Sliderbody artefacts 

### ISSUE THREAD

**Issue by OninDynamics** [ppy/osu#23171]
### Type

Cosmetic

### Bug description

Slider body rendering behaves... strangely on this map: https://osu.ppy.sh/beatmapsets/705788#osu/1492654. Usually the longer ones, and sometimes snaking out sliders also leave out similar trails. Difficult to describe, so refer to video ig

Only happens in non-legacy OpenGL. (Veldrid / OpenGL)
Only happens when storyboards are enabled. 
Only when background dim is set to 100%. (Discovered a bit later, so not in logs) 
Beatmap settings to toggle beatmap colours and skins will clear the artefacts, but not prevent it.

Observed on Arch Linux, will test on Windows.

### Screenshots or videos

https://youtu.be/hXDtfDfHaFI

### Version

osu! 2023.403.1-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/11184836/database.log)
[input.log](https://github.com/ppy/osu/files/11184837/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/11184838/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/11184839/network.log)
[performance-audio.log](https://github.com/ppy/osu/files/11184840/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/11184841/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/11184842/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/11184843/performance-update.log)
[performance.log](https://github.com/ppy/osu/files/11184844/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11184845/runtime.log)

**Comment by Gabixel** [comment]
This looks like a duplicate of #23046

**Comment by OninDynamics** [comment]
Yep, it indeed is. Looks like I didn't look around hard enough...
